docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,841 @@
+
+Now that you've gotten the data from the user, it's time to respond to them.
+
+## Responding to an Interaction
+
+Interactions--both receiving and responding--are webhooks under the hood. So responding to an Interaction is just like sending a webhook request!
+
+When responding to an interaction received **via webhook**, your server can simply respond to the received `POST` request. You'll want to respond with a `200` status code (if everything went well), as well as specifying a `type` and `data`, which is an object with parameters the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook):
+
+```py
+@app.route('/', methods=['POST'])
+def my_command():
+    if request.json["type"] == 1:        
+        return jsonify({
+            "type": 1
+        })
+    
+    else:
+        return jsonify({
+            "type": 4,
+            "data": {
+                "tts": False,
+                "content": "Congrats on sending your command!",
+                "embeds": [],
+                "allowed_mentions": []
+            }
+        })
+```
+
+If you are receiving Interactions over the gateway, you will **also need to respond via HTTP**. Responses to Interactions **are not sent as commands over the gateway**.
+
+To respond to a gateway Interaction, make a `POST` request like this. `interaction_id` is the unique id of that individual Interaction from the received payload. `interaction_token` is the unique token for that interaction from the received payload. **This endpoint is only valid for Interactions received over the gateway. Otherwise, respond to the `POST` request.**
+
+```py
+url = "https://discord.com/api/v8/interactions/<interaction_id>/<interaction_token>/callback"
+
+json = {
+    "type": 4,
+    "data": {
+        "content": "Congrats on sending your command!"
+    }
+}
+r = requests.post(url, json=json)
+```
+
+> info
+> Interaction `tokens` are valid for **15 minutes** and can be used to send followup messages.
+
+## Followup Messages
+
+Sometimes, your bot will want to send followup messages to a user after responding to an interaction. Or, you may want to edit your original response. You can use the following endpoints to do so:
+
+- `PATCH /webhooks/<application_id>/<interaction_token>/messages/@original` to edit your initial response to an Interaction
+- `DELETE /webhooks/<application_id>/<interaction_token>/messages/@original` to delete your initial response to an Interaction
+- `POST /webhooks/<application_id>/<interaction_token>` to send a new followup message
+- `PATCH /webhooks/<application_id>/<interaction_token>/messages/<message_id>` to edit a message sent with that `token`
+
+> info
+> Interactions webhooks share the same rate limit properties as normal webhooks
+
+Interaction tokens are valid for **15 minutes**, meaning you can respond and update to a Slash Command within that amount of time.
+
+## Security and Authorization
```

Comment:
To prevent a replay attack (hah, as if that would happen over https), should we validate the timestamp is fewer than `n` seconds old? Is there a `n` such that Discord guarantees the request will arrive within said `n` seconds?

Reply:
@night would probably be a better answer for that. I think that in doing the signature validation, you should be guaranteed that it's fine, but I'm not the security expert 😄